apps/notifications/README.md

```diff
@@ -17,6 +17,10 @@ Alternatively you can manually build the app with `lerna` and copy the built fil
 ```bash
 # Builds files and places them in `apps/notifications/dist`
 npx lerna run build --scope="@automattic/notifications"
+
+# Or directly with yarn:
+cd apps/notifications
+yarn build
```

Comment:
I think this could replace the lerna command, personally I don't think we need to document both ways (will need to update the "Alternatively you can manually build the app with `lerna` and copy the built files to your sandbox" sentence above too)

Reply:
sounds good! 👍